### PR TITLE
Feature/22_LOGIN_VIEW

### PR DESCRIPTION
**Pull request by Symphonie234**
- Added CSS to the login view.
- Created a new CSS file called "Login.css" to implement a new technique in using CSS.
- Also used the same tactic above but for the page title.
- Google Login is functional.

### CODE REVIEW

File: composer.lock
Comment (ChronoDevs) [inline composer.lock:1]: don't push composer.lock

File: resources/views/auth/login.blade.php [hunk 1]
```diff
@@ -1,9 +1,10 @@
 @extends('layouts.app')
-
+@section('title', 'Login')
+@section('css', 'login.css')
 @section('content')
 <div class="container">
     <div class="row justify-content-center">
-        <div class="col-md-8">
+        <div class="col-md-5">
             <div style="display: flex; justify-content: center; align-items: center;">
```
Comment (ChronoDevs) [inline resources/views/auth/login.blade.php]: don't add style inside the html tag

File: public/js/scripts.js [hunk 2]
```diff
@@ -0,0 +1,10 @@
+document.addEventListener("DOMContentLoaded", function() {
```
Comment (ChronoDevs) [inline public/js/scripts.js]: use single quote

File: resources/views/auth/login.blade.php [hunk 3]
```diff
@@ -1,9 +1,10 @@
...
                 <div class="card-body" id="card-body">
                     <div class="intro" style="text-align: center;">
```
Comment (ChronoDevs) [inline resources/views/auth/login.blade.php]: don't add style inside the html tag

File: resources/views/auth/login.blade.php [hunk 4]
```diff
@@ -19,39 +20,50 @@
                     </div>
                     <br>
                     <div class="social-login" style="text-align: center;">
```
Comment (ChronoDevs) [inline resources/views/auth/login.blade.php]: don't add style inside the html tag

File: composer.lock
Comment (Symphonie234) [reply composer.lock:1]: Understood

File: resources/views/auth/login.blade.php [hunk 5]
```diff
@@ -19,39 +20,50 @@
...
-                        <button class="btn btn-primary" id="fb-button">Continue with Facebook</button>
-                        <button class="btn btn-danger" id="google-button">Continue with Google</button>
+                        <a href="{{ route('login.facebook') }}" class="btn btn-primary" id="facebook-button">
+                            <img src="images/facebook_icon.png" alt="Facebook Icon" class="button-icon">
+                            Continue with Facebook
+                        </a>
+                    </div>
+                    <div class="social-login" style="text-align: center;">
```
Comment (ChronoDevs) [inline resources/views/auth/login.blade.php]: don't add style inside the html tag

File: resources/views/auth/login.blade.php (hunk 1 above)
Comment (Symphonie234) [reply resources/views/auth/login.blade.php]: Understood

File: config/services.php [hunk 6]
```diff
@@ -30,4 +30,15 @@
         'region' => env('AWS_DEFAULT_REGION', 'us-east-1'),
     ],
 
+    'facebook' => [
```
Comment (ChronoDevs) [inline config/services.php:33]: commit env,example also

**Comment by Symphonie234** [comment]
- removed styling in the HTML
- changed double quotes (" ") in the js folder into single quotes (' ')
- removed the composer.lock
- added content in the .env.example file

File: database/migrations/2014_10_12_000000_create_users_table.php [hunk 7]
```diff
@@ -26,9 +26,11 @@ public function up()
             $table->integer('zip_code')->nullable();
             $table->text('address')->nullable();
             $table->string('email')->unique();
+            $table->string('provider_id')->nullable();
+            $table->string('avatar')->nullable();
             $table->timestamp('email_verified_at')->nullable();
             $table->string('password');
-            $table->rememberToken();
+            $table->rememberToken()a;
```
Comment (ChronoDevs) [inline database/migrations/2014_10_12_000000_create_users_table.php]: a is created?

**Comment by Symphonie234** [comment]
- fixed the extra 'a' in the users table.

**Pull request merged**